docs/reference/filebeat/migrate-to-filestream.md

```diff
@@ -162,23 +169,20 @@ filebeat.inputs:
      enabled: true
    paths:
      - /var/log/java-exceptions*.log
-
```

Comment:
[Nit]

That's probably a personal preference, but those blank lines make a visual separation of each input block, making it easier to read the configuration.